packages/clerk-js/src/core/auth/AuthCookieService.ts

```diff
@@ -188,20 +191,20 @@ export class AuthCookieService {
    * The below methods are used to determine whether or not an unfocused tab can be responsible
    * for setting the session cookie. A session cookie should only be set by a tab who's selected
    * organization matches the session's active organization. By storing the active organization
-   * ID in local storage, we can check the value across tabs. If a tab's organization ID does not
-   * match the value in local storage, it is not responsible for updating the session cookie.
+   * ID in a cookie, we can check the value across tabs. If a tab's organization ID does not
+   * match the value in the active org cookie, it is not responsible for updating the session cookie.
    */
 
   public setActiveOrganizationInStorage() {
     if (this.clerk.organization?.id) {
-      localStorage.setItem('clerk_active_org', this.clerk.organization.id);
+      this.activeOrgCookie.set(this.clerk.organization.id);
     } else {
-      localStorage.removeItem('clerk_active_org');
+      this.activeOrgCookie.remove();
     }
   }
 
   private isCurrentOrganizationActive() {
-    const activeOrganizationId = localStorage.getItem('clerk_active_org');
+    const activeOrganizationId = this.activeOrgCookie.get();
```

Comment:
❓ Do you think we need to be backwards compatible here? Like, try to check localStorage if there is no cookie? (inside a try/catch of course)

Reply:
I don't think so, as `setActiveOrganizationInStorage` gets called on load, so it should self-update 👍